webapp/views/wpt-results.js

```diff
@@ -1045,20 +1045,24 @@ class WPTResults extends Pluralizer(WPTColors(WPTFlags(PathInfo(LoadingState(Tes
     }
   }
 
-  handleSelectMetadata(index, test) {
+  handleSelectMetadata() {
     return (e) => {
+      // Prevent accidentally modifying these captured parameters in the closure.
+      const [index, test] = arguments;
```

Comment:
We should capture the arguments in the parent function instead.

```suggestion
    // Prevent accidentally modifying these captured parameters in the closure.
    const [index, test] = arguments;
    return (e) => {
```